Add medallia feeback button to remaining templates

aubreyarcangel opened the pull request:
## Description
This PR addresses [#41572](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41572) and is a continuation of the effort to [add the Medallia feedback button sitewide.
](https://github.com/department-of-veterans-affairs/content-build/pull/810)

Medallia feedback button added to the templates below:

|template                                                    |example url                                                         |
|------------------------------------------------------------|--------------------------------------------------------------------|
|src/site/layouts/story_listing.drupal.liquid                |/pittsburgh-health-care/stories/                                    |
|src/site/layouts/publication_listing.drupal.liquid          |/outreach-and-events/outreach-materials/                            |
|src/site/layouts/press_releases_listing.drupal.liquid       |/wilkes-barre-health-care/news-releases/                            |
|src/site/layouts/press_release.drupal.liquid                |/wilkes-barre-health-care/news-releases/veteran-transitions/        |
|src/site/layouts/person_profile.drupal.liquid               |/tuscaloosa-health-care/staff-profiles/june-griffith/               |
|src/site/layouts/news_story.drupal.liquid                   |/pittsburgh-health-care/stories/treating-veterans-with-long-covid/  |
|src/site/layouts/locations_listing.drupal.liquid            |/tuscaloosa-health-care/locations/                                  |
|src/site/layouts/leadership_listing.drupal.liquid           |/wilkes-barre-health-care/about-us/leadership/                      |
|src/site/layouts/health_services_listing.drupal.liquid      |/tuscaloosa-health-care/health-services/                            |
|src/site/layouts/q_a.drupal.liquid                          |/resources/can-i-plan-ahead-for-my-burial-in-a-va-national-cemetery/|
|src/site/layouts/office.drupal.liquid                       |/outreach-and-events/                                               |
|src/site/layouts/vamc_system_billing_insurance.drupal.liquid|/new-york-harbor-health-care/billing-and-insurance/                 |
|src/site/layouts/event_listing.drupal.liquid                       |/pittsburgh-health-care/events/                                              |
 
## Testing done
Viewed locally and on review build

## Screenshots
before:
<img width="1344" alt="Screen Shot 2022-09-27 at 10 24 09 AM" src="https://user-images.githubusercontent.com/11021491/192553284-2c2d56e7-75fe-4614-bc5e-f7d741e24776.png">

after:
<img width="1399" alt="Screen Shot 2022-09-27 at 10 23 41 AM" src="https://user-images.githubusercontent.com/11021491/192553346-da82e404-cc01-4853-a5a9-dbe0ba2cbca0.png">

before:
<img width="1215" alt="Screen Shot 2022-09-27 at 10 35 25 AM" src="https://user-images.githubusercontent.com/11021491/192557571-1c181430-efeb-49bb-9a55-34b6ffe2a783.png">

after:
<img width="1285" alt="Screen Shot 2022-09-27 at 10 35 07 AM" src="https://user-images.githubusercontent.com/11021491/192557530-80814e75-9d15-4988-847b-dc0f53756a38.png">

before:
<img width="1307" alt="Screen Shot 2022-09-27 at 11 07 41 AM" src="https://user-images.githubusercontent.com/11021491/192564401-019bb7a0-9323-4b7c-986c-f9a0ff0e0f96.png">

after:
<img width="1331" alt="Screen Shot 2022-09-27 at 11 08 02 AM" src="https://user-images.githubusercontent.com/11021491/192564428-05a77785-3089-4a2d-8fc7-809bb784eb94.png">

before:
<img width="1284" alt="Screen Shot 2022-09-27 at 11 30 31 AM" src="https://user-images.githubusercontent.com/11021491/192569837-aa7aa8ac-9c2e-4b22-addf-eeb0034deed7.png">

after:
<img width="1226" alt="Screen Shot 2022-09-27 at 11 30 03 AM" src="https://user-images.githubusercontent.com/11021491/192569912-b28523a6-9b14-4d08-a907-b7379b8b9a3d.png">


## Acceptance criteria
- [X] Feedback button is added to the bottom of main content 

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

maxx1128 reviewed (commented):
The code checks out, but the ticket description says to update `vamc_system_register_for_care.drupal.liquid` and it's not changed. `event_listing` is also updated even though it's on the list. Was this a change added to the requirements later or a mistake in the files changes?

aubreyarcangel commented:
> The code checks out, but the ticket description says to update `vamc_system_register_for_care.drupal.liquid` and it's not changed. `event_listing` is also updated even though it's on the list. Was this a change added to the requirements later or a mistake in the files changes?

@maxx1128 Good catch, thank you! It was an oversight on my end. I added `event_listing` to the list above and added the feedback button to the `vamc_system_register_for_care.drupal.liquid` template.
